### PR TITLE
Remove clikt

### PR DESCRIPTION
**Pull request by tcasstevens**

### CODE REVIEW

**Comment by lynnjo** [comment]
The test files in the cli folder (MergeGvcfsTest, MutateProteinTest, ValidateGvcfsTest) were removed as those clikt files were removed and this is correct. But shouldn't the individual test cases from those files be re-added as junit tests with new files under util/MutateProtienUtilsTest.kt, util/ValidateGvcfUtilsTest.kt and util/MergeGVCFsUtil.kt?

**Comment by tcasstevens** [comment]
> The test files in the cli folder (MergeGvcfsTest, MutateProteinTest, ValidateGvcfsTest) were removed as those clikt files were removed and this is correct. But shouldn't the individual test cases from those files be re-added as junit tests with new files under util/MutateProtienUtilsTest.kt, util/ValidateGvcfUtilsTest.kt and util/MergeGVCFsUtil.kt?

I'm moving those to the biokotlin-tools repository

**Comment by tcasstevens** [comment]
> The test files in the cli folder (MergeGvcfsTest, MutateProteinTest, ValidateGvcfsTest) were removed as those clikt files were removed and this is correct. But shouldn't the individual test cases from those files be re-added as junit tests with new files under util/MutateProtienUtilsTest.kt, util/ValidateGvcfUtilsTest.kt and util/MergeGVCFsUtil.kt?

@lynnjo Ok, I added those back to the biokotlin repository

**Comment by lynnjo** [comment]
Thank you @tcasstevens !

**Review by lynnjo — APPROVED** [review]

File: src/main/kotlin/biokotlin/util/ValidateGVCFsUtils.kt [hunk 1]
```diff
@@ -0,0 +1,110 @@
+package biokotlin.util
+
+import biokotlin.genome.fastaToNucSeq
+import kotlinx.coroutines.runBlocking
+import org.apache.logging.log4j.LogManager
+import java.io.File
+
+private val myLogger = LogManager.getLogger("biokotlin.util.ValidateGVCFsUtils")
+
+fun validateGVCFs(inputDir: String, outputDir: String, referenceFile: String, correct: Boolean = false) {
```
Comment (zrm22) [inline src/main/kotlin/biokotlin/util/ValidateGVCFsUtils.kt:10]: Consider a Companion object here

**Comment by zrm22** [comment]
The Unit tests have a lot of commented out code.  I would recommend removing it.

**Review by zrm22 — APPROVED** [review]

File: src/main/kotlin/biokotlin/util/MutateProteinsUtils.kt [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
+enum class TypeMutation {
+    DELETION, POINT_MUTATION, INSERTION
+}
+
+val proteinLetters = arrayOf(
+    'R', 'H', 'K', 'D', 'E', 'S', 'T', 'N', 'Q', 'C', 'G', 'P', 'A', 'V', 'I', 'L', 'M', 'F', 'Y', 'W'
+)
+
+val numProteinLetters = proteinLetters.size
```
[the remark this answers is not in the document]
Comment (tcasstevens) [reply src/main/kotlin/biokotlin/util/MutateProteinsUtils.kt:29]: Do you mean a singleton object?  Like this? Companion objects can't live outside a class?

object MutateProteinsUtils {

}

[the remark this answers is not in the document]
Comment (zrm22) [reply src/main/kotlin/biokotlin/util/MutateProteinsUtils.kt:29]: We do this in phgv2 a lot.  I think you are right that you need to make a MutateProteinsUtils class first then a companion object within it holding your functions.

[the remark this answers is not in the document]
Comment (tcasstevens) [reply src/main/kotlin/biokotlin/util/MutateProteinsUtils.kt:29]: Ok, I changed these functions to be in a Singleton Object

File: src/test/kotlin/biokotlin/util/MergeGVCFsTest.kt [hunk 3]
```diff
@@ -51,13 +48,11 @@ class MergeGVCFsTest {
         // --input-dir /Users/tmc46/git/biokotlin/data/test/merge_gvcfs_input/
         // --output-file /Users/tmc46/git/biokotlin/data/test/merge_gvcfs_output.vcf
 
-        val result = MergeGVCFs().test(
-            "--input-dir $gvcfInputDir --output-file $testMergedOutputFile"
-        )
+        // val result = MergeGVCFs().test(
```
[the remark this answers is not in the document]
Comment (tcasstevens) [reply src/test/kotlin/biokotlin/util/MergeGVCFsTest.kt]: Yes, I removed those comments from the unit tests.

**Pull request merged**